Extensions/xkit_preferences.js

```diff
@@ -1405,33 +1405,41 @@ XKit.extensions.xkit_preferences = new Object({
 
 			$("#xkit-extensions-panel-right-inner").html('<div id="xkit-extension-panel-no-settings">Updating...</div>');
 
-			if (typeof XKit.extensions.xkit_updates === "undefined" || typeof XKit.extensions.xkit_updates.update === "undefined") {
+			if (typeof XKit.extensions.xkit_updates === "undefined" ||
+				typeof XKit.extensions.xkit_updates.update === "undefined") {
+
 				XKit.window.show("Can't update",
-					'It looks like "XKit Updates" extension is missing or not working properly. It is highly recommended that you reset XKit.', "error",
+					'It looks like "XKit Updates" extension is missing or not working properly. '+
```

Comment:
Right, but we're touching these lines anyways.

I definitely agree though, we need to run through our copy all at once so we're not duplicating work.